test/testdata/goldens/embedding/embedding_test/unexpected exception writes the right message.txt

```diff
@@ -0,0 +1,371 @@
+# GENERATED BY: test/embedding/embedding_test.dart
+
+$ tool/test-bin/pub_command_runner.dart pub fail
+[E] Bad state: Pub has crashed
+[E] tool/test-bin/pub_command_runner.dart 32:5   ThrowingCommand.runProtected
+[E] package:pub/src/command.dart 173:45          PubCommand.run.<fn>
+[E] package:pub/src/command.dart 173:33          PubCommand.run.<fn>
+[E] dart:async                                   new Future.sync
+[E] package:pub/src/utils.dart 109:12            captureErrors.wrappedCallback
+[E] dart:async                                   runZonedGuarded
+[E] package:pub/src/utils.dart 126:5             captureErrors
+[E] package:pub/src/command.dart 173:13          PubCommand.run
+[E] package:args/command_runner.dart 209:27      CommandRunner.runCommand
+[E] tool/test-bin/pub_command_runner.dart 62:24  Runner.runCommand
+[E] tool/test-bin/pub_command_runner.dart 53:20  Runner.run
+[E] tool/test-bin/pub_command_runner.dart 67:29  main
+[E] This is an unexpected error. See the full transcript in
+[E] 
+[E]     /tmp/dart_test_PRKENG/cache/log/pub_log.txt
+[E] 
+[E] and attach the transcript in an issue on https://github.com/dart-lang/pub/issues/new
+
+-------------------------------- END OF OUTPUT ---------------------------------
+
+File containing information about the latest pub run.
+Before making this file public, make sure to remove any sensitive information!
+
+Pub version: 0.1.2+3
+FLUTTER_ROOT: <not set>
+Command: /usr/local/google/home/sigurdm/projects/dart-sdk/sdk/out/ReleaseX64/dart-sdk/bin/dart
```

Comment:
I'm guessing this golden test isn't very portable? :D

Reply:
OOps :)